bin/dasht-query-line

```diff
@@ -259,9 +259,12 @@ dasht-docsets "$@" | while read -r docset; do
 
       # resolve URL to filesystem location
       $3 = file_url $3
-    }
 
-    /./ # reject any empty lines from input
+      # print TSV line
+      printf "%s\t%s\t%s\t%s\n", result["name"], docset, \
```

Comment:
I would add parentheses to the `printf()` call and list each argument on its own line.